app/assets/stylesheets/countries.scss

```diff
@@ -0,0 +1,11 @@
+#container {
```

Comment:
similarly here - note how some of these styles are duplicated now. This should all be consolidated into a single stylesheet.